src/pages/DriverDash.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+class DriverDash extends React.Component {
```

Comment:
Should we rename DriverDash to something else? Maybe just dashboard? This doesn't need to be addressed with this PR but something to think about bc user tests revealed that people were kinda confused on what it meant

Reply:
We can just rename it to that yeah. The button could be renamed to Dashboard, but we would need to disable the button for Riders (i.e. separate home page using your PrivateRoute).